aten/src/ATen/cuda/CUDAFuture.cpp

```diff
@@ -40,19 +40,61 @@ std::vector<std::reference_wrapper<const at::DataPtr>> extractDataPtrs(
   return data_ptrs;
 }
 
+std::vector<c10::DeviceIndex> getDevicesOfDataPtrs(
+    const std::vector<std::reference_wrapper<const at::DataPtr>>& data_ptrs) {
+  std::vector<bool> isCudaDeviceUsed(c10::cuda::device_count(), false);
+  for (const at::DataPtr& data_ptr : data_ptrs) {
+    if (data_ptr.device().is_cuda()) {
+      isCudaDeviceUsed[data_ptr.device().index()] = true;
+    }
+  }
+  std::vector<c10::DeviceIndex> device_indices;
```

Comment:
camel naming?

Reply:
I'm seeing a lot of inconsistency between snake case and camel case, hence I never know what to do. Is there some convention we're following/enforcing?

Reply:
I usually try to keep the same naming convention within the same file.